README.md

```diff
@@ -149,13 +150,13 @@ $ balena push DebugApp
 Any device owned by a customer automatically allows access by that user via
 either the WebTerminal (in the device's Dashboard page), or the balena CLI
 via `balena ssh <uuid>`. However, for a support agent to gain access to a device
-that isn't owned by them, a user or collaborator that does have access must
+that isn't owned by them, a user that does have access must
 grant it explicitly.
 
 #### 1.1 Granting Support Access to a Support Agent
 
 A user can grant access to support agents by selecting the device from the
-Dashboard, and then selecting the 'Actions' tab in the left-hand sidepanel.
+Dashboard, and then selecting the 'Actions' tab in the left-hand side panel.
```

Comment:
```suggestion
Dashboard, and then selecting the 'Actions' tab in the sidebar.
```

Side bar naturally is the word for a "Panel on the left" Since no one does panels on the right for web applications except in mobiles.

Reply:
SGTM @vipulgupta2048 👍